### PR TITLE
docs:2253 - Updated the docs for the the Search and Feedback

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 2253

### 

# Explain the update

Included docs for search and feedback with API references
## Does this PR introduce a breaking change

No

## Screenshots
<details>
<summary>View Screenshots</summary>

image
image
image
image

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?

### CODE REVIEW

File: packages/doc-spa/.env.example [hunk 1]
```diff
@@ -1,5 +1,3 @@
-REACT_APP_OPCBASE_API_BASE_PATH=
```
Comment (akhilmhdh) [inline packages/doc-spa/.env.example:1]: Why is this removed. This is used env variable

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

File: packages/doc-spa/.env.example (hunk 1 above)
Comment (riginoommen) [reply packages/doc-spa/.env.example]: Oops my bad restored it

**Review by akhilmhdh — DISMISSED** [review]

**Review by deshmukhmayur — APPROVED** [review]

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]
@riginoommen can you check why the CI is failing?

**Review by hybridx — DISMISSED** [review]

**Comment by riginoommen** [comment]
> @riginoommen can you check why the CI is failing?

Pipelines are passing now. and merging
